src/graphdb.js

```diff
@@ -74,52 +121,108 @@ function graphDBInferenceSchema (graphbSchema, addMutations) {
         node.properties.forEach(property => {
             if (property.name == 'id')
                 r+= `\tid: ID\n`;
+            else if (checkInvalidChar(property.name)) {
+                r+= `\t${replaceCleanseLabel(property.name)}: ${property.type} @alias(property: "${property.name}")\n`;
+            }
             else
                 r+= `\t${property.name}: ${property.type}\n`;
         });
         
         let edgeTypes = [];
         gdbs.edgeStructures.forEach(edge => {            
             edge.directions.forEach(direction => {
-                let fromCase = toPascalCase(direction.from);
-                let toCase = toPascalCase(direction.to);
-                let edgeCase = toPascalCase(edge.label);
+                invalidNode = checkInvalidChar(node.label);
+                let invalidDirFrom = checkInvalidChar(direction.from);
+                let invalidDirTo = checkInvalidChar(direction.to);
+                let invalidEdge = checkInvalidChar(edge.label);
+
+                let nodeCase = invalidNode ? replaceCleanseLabel(node.label) : node.label;
```

Comment:
The node label is already processed above this loop so you shouldn't have to do it again here, just reference the processed node variable name `nodeCase`.